sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/tests/Tests/DesktopVirtualizationUnitTests.cs

```diff
@@ -0,0 +1,549 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.DesktopVirtualization.Models;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.DesktopVirtualization.Tests.Tests
```

Comment:
It's better to separate the tests and update the project structure for the maintenance, we have a test guide [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md).

Reply:
Refactored